src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/Migrations/00000000000000_CreateIdentitySchema.cs

```diff
@@ -1,14 +1,16 @@
-﻿using Microsoft.EntityFrameworkCore.Metadata;
+﻿using System;
```

Comment:
in all migration files add
`// This file isn't generated, but this comment is necessary to exclude it from StyleCop analysis.
// <auto-generated/>`
on the top and remove pragma warnings